docs/devices/ZNXNKG02LM.md

```diff
@@ -13,90 +13,100 @@ pageClass: device-page
 
 # Xiaomi ZNXNKG02LM
 
-|     |     |
-|-----|-----|
-| Model | ZNXNKG02LM  |
-| Vendor  | [Xiaomi](/supported-devices/#v=Xiaomi)  |
-| Description | Aqara knob H1 (wireless) |
-| Exposes | battery, voltage, action, operation_mode, action_rotation_angle, action_rotation_angle_speed, action_rotation_percent, action_rotation_percent_speed, action_rotation_time, linkquality |
-| Picture | ![Xiaomi ZNXNKG02LM](https://www.zigbee2mqtt.io/images/devices/ZNXNKG02LM.jpg) |
-
+|             |                                                                                                                                                                                         |
+| ----------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Model       | ZNXNKG02LM                                                                                                                                                                              |
```

Comment:
Please keep the formatting here, this is generated and will be reversed automatically with the next release.

Reply:
yes i edited